src/main/kotlin/RockPaperSmash/Driver.kt

```diff
@@ -0,0 +1,1000 @@
+
+    /**
+     * Directories for all titles.
+     */
+    private val TITLE_LOC = arrayOf("Assets/DisplayScreens/Title/Title1.txt", "Assets/DisplayScreens/Title/Title2.txt", "Assets/DisplayScreens/Title/Title3.txt")
+
+    /**
+     * Copies of every character for easy selection. Likely to be replaced by strings later.
+     */
+    private val CHAR_LIST = arrayOf(Marth("NULL"), Fox("NULL"), Falco("NULL"), CaptainFalcon("NULL"))
+
+    /**
+     * Copies of every stage, likely to be replaced later.
+     */
+    private val STAGE_LIST = arrayOf<Stage>(Battlefield())
+
+    private val CPU_IDENTIFIER = "CPU"
+
+    /**
+     * Names of player 1 and player 2.
+     */
+    private lateinit var mP1Name: String
+    private lateinit var mP2Name: String
+
+    private val ATTACK_STRING = ", what move do you want to make? (a)ttack, (g)rab, (s)hield\n> "
+
+    private val TMP_DIR = FileReader.tmpDir
+
+    /**
+     * Allows choice of a [Stage].
+     *
+     * @return Chosen stage.
+     */
+    // Test validity of player choice for a character
+    // Only go here if the player chose a valid Stage
+    // Should never reach here
+    // Stage becomes Battlefield if some impossible flaw in logic causes this to happen
+    val field: Stage
+        get() {
+            var input: String
+            var response: String
+            var goodPick = false
+            val sc = Scanner(System.`in`)
+            println()
+            while (!goodPick) {
+                print("Stage list: ")
+                for (st in STAGE_LIST) {
+                    print("\"" + st.stageName + "\" ")
+                }
+                println()
+                print("What stage would you like to pick?\n> ")
+                input = sc.nextLine().toLowerCase().trim({ it <= ' ' })
+
+                var count = 0
+                for (st in STAGE_LIST) {
+                    if (st.stageName.toLowerCase() == input) {
+                        goodPick = true
+                        break
+                    }
+                    ++count
+                }
+                if (!goodPick) {
+                    println("Sorry, I don't recognize that stage.")
+                    println()
+                }
+                if (goodPick) {
+                    animate(FileResource(STAGE_LIST[count].fileLoc))
+                    print("Is \"$input\" good?\n> ")
+                    response = sc.nextLine()
+                    response = response.toLowerCase().trim({ it <= ' ' })
+                    for (comparator in POS_RESPONSES) {
+                        if (response == comparator) {
+                            return STAGE_LIST[count]
+                        }
+                    }
+                    goodPick = false
+                }
+            }
+            println("Fatal choosing error. Defaulting to Battlefield.")
+            return Battlefield()
+        }
+
+    @JvmStatic
+    fun main(args: Array<String>) {
```

Comment:
remove args from main

Reply:
No such luck. You need to include args. Otherwise, it cannot find the main function.